test/test-util/getBalanceAndApprove.ts

```diff
@@ -37,12 +37,6 @@ export const getBalanceAndApprove = async (
     await (
       await aliceTokenIn.approve(approveTarget, constants.MaxUint256)
     ).wait();
-
```

Comment:
Hmm yeah that implementation of approval check is wrong, since for some tokens like USDT, max approval is not uint256 max bc 6 decimals. I wasn't sure if it was the case for other tokens so I just removed it.